### DISCUSSION TITLE
Can helmfile diff be enabled to do a 3 way merge

### DISCUSSION THREAD

**Discussion by har26399** [helmfile/helmfile#559] [Questions]
Since `v3.3.0` of `helm diff`, `helm3` supports three way merges. I'm on version `v0.147.0` of `helmfile` but I can't seem to enable three way merges for `helmfile diff`. doesn't helmfile diff use helm diff under the hood? am I understanding this incorrectly?

**Comment by yxxhero** [comment]
@har26399 You can try to use env var to enable three way merges for helmfile diff. please see helm-diff docs.
